koditools/pidgin.py

```diff
@@ -8,43 +9,50 @@
 from .kodiclient import KodiClient
 from . import utils
 
+
 def toText(val):
     if isinstance(val, unicode):
         val = val.encode('utf8')
     return val
 
+
 def html2text(html):
     output = StringIO()
     writer = DumbWriter(output)
     p = HTMLParser(AbstractFormatter(writer))
     p.feed(toText(html))
-    return  toText(output.getvalue())
+    return toText(output.getvalue())
+
 
 class Forwarder(object):
     def __init__(self, host):
         if not host:
             cfg = utils.getConfigFile()
-            if not cfg.has_section('server') or not cfg.has_option('server', 'host'):
-                raise ValueError('Not host configured')
+            if not cfg.has_section('server') or \
+               not cfg.has_option('server', 'host'):
```

Comment:
This could possibly be replaced with something like `if not (cfg.has_section(...) and cfg.has_option(...)):`, but didn't check whether that would break the >79 character rule again.